nuxeo-layout.html

```diff
@@ -84,8 +84,34 @@
       validate: function() {
         if (this.element && ('function' === typeof this.element.validate)) {
           return this.element.validate();
+        } else {
+          // workaroud for https://github.com/PolymerElements/iron-form/issues/218, adapted from iron-form.html
+          var valid = true;
+          if (this.element) {
+            var elements = this._getValidatableElements(this.element.root);
```

Comment:
nit: leverage Array.some()
`return !this._getValidatableElements(this.element.root).some(function(e) { return !el.validate(); });`

Reply:
👍

Reply:
As a matter of fact, using `some` would stop at the first invalid field, but perhaps it would be better to keep highlighting all invalid fields.